controllers/upgrade.go

```diff
@@ -79,6 +89,7 @@ func (r *RollingUpgradeReconciler) ReplaceNodeBatch(rollingUpgrade *v1alpha1.Rol
 		for _, target := range batch {
 			_ = target
 			// Add in-progress tag
+			r.SetInstanceTag(rollingUpgrade, target, instanceStateTagKey, inProgressTagValue)
```

Comment:
Can we just call `r.Auth.TagEC2instance` directly and avoid the extra method? we should probably also handle the error condition